src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp

```diff
@@ -185,7 +191,9 @@ namespace OpenMS
     double delta_int_apex = delta_int*delta_rt_apex/delta_rt;
 
     avg_noise_level = intensity_min + delta_int_apex;
```

Comment:
so basically what you are calculating here is the noise level *at the peak apex* - so this calculation is probably best for correcting peak apex calculations. I am not so sure how good it is for calculating peak area corrections, especially if the peak apex is not centered. Practically I dont think it makes a big difference but to me it seems that what I am doing currently is better for the area correction while what you are doing is better for the peak apex intensity correction

Reply:
Yes exactly.  For the area version I am walking the line from best left to best right and summing the intensities.